apitools/org.eclipse.pde.api.tools/src/org/eclipse/pde/api/tools/internal/ApiAnalysisApplication.java

```diff
@@ -189,6 +190,31 @@ public Object start(IApplicationContext context) throws Exception {
 		}
 	}
 
+	private static void printMarker(IMarker marker, String type) {
+		String path = getFullPath(marker);
+		String file = marker.getResource().getName();
+		int lineNumber = marker.getAttribute(IMarker.LINE_NUMBER, -1);
+		String message = marker.getAttribute(IMarker.MESSAGE, "").trim(); //$NON-NLS-1$
```

Comment:
strip() removes all kind of whitespaces.

Reply:
I don't think that it makes any difference here.